Minio.Examples/Cases/SetLegalHold.cs

```diff
@@ -0,0 +1,58 @@
+/*
+ * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Minio.DataModel;
+
+namespace Minio.Examples.Cases
+{
+    class SetLegalHold
+    {
+        // Enable Legal Hold and then Check if Legal Hold is Enabled on a bucket
+        public async static Task Run(MinioClient minio,
+                                     string bucketName = "my-bucket-name",
+                                     string objectName = "my-object-name",
+                                     string versionId = "")
+        {
+            try
+            {
+                Console.WriteLine("Running example for API: SetLegalHold, enable legal hold");
+
+                SetObjectLegalHoldArgs args = new SetObjectLegalHoldArgs(bucketName, objectName)
+                                                    .WithVersionID(versionId)
+                                                    .WithLegalHoldON();
+                await minio.SetLegalHoldConfigurationAsync(args);
+                Console.WriteLine("Legal Hold status for bucket " + bucketName + " object " + objectName + 
+                            (string.IsNullOrEmpty(versionId)?" " : " with version id " + versionId + " ") + 
+                            " set to ON." );
+                Console.WriteLine();
+                Console.WriteLine("Running example for API: SetLegalHold, disable legal hold ");
```

Comment:
you don't need example for disabling legalhold

Reply:
Removed.